rules/config

```diff
@@ -153,7 +153,7 @@ INCLUDE_DHCP_RELAY = y
 INCLUDE_DHCP_SERVER ?= n
 
 # INCLUDE_P4RT - build docker-p4rt for P4RT support
-INCLUDE_P4RT = n
+INCLUDE_P4RT = y
```

Comment:
Is this change intentional, to build the P4RT docker by default? Was it brought up/discussed somewhere?

Reply:
It looked like it had been turned off, since a build failure was encountered at some point.   I wasn't able to reproduce the build failure, so I thought it could be turned back on.

Reply:
@dgsudharsan I see that #16343 disabled the P4RT build by default due to some build issues. Are you still seeing these issues?

Reply:
We have not enabled it since disabling. I would recommend fixing UT issues I raised before reenabling

Reply:
Thanks for the comments.  I went ahead and reverted this particular change, since it's unrelated to upgrading to Bookworm.